src/test/tests/scanGraphLogic.test.ts

```diff
@@ -0,0 +1,77 @@
+import { assert } from 'chai';
+import { IGraphRequestModel } from 'jfrog-client-js';
+
+import { ConnectionManager } from '../../main/connect/connectionManager';
+import { GraphScanLogic } from '../../main/scanLogic/scanGraphLogic';
+import { RootNode } from '../../main/treeDataProviders/dependenciesTree/dependenciesRoot/rootTree';
+import { DependenciesTreeNode } from '../../main/treeDataProviders/dependenciesTree/dependenciesTreeNode';
+import { createDependency } from './utils/treeNodeUtils.test';
+import { PackageType } from '../../main/types/projectType';
+
+describe('GraphScanLogic Tests', async () => {
```

Comment:
I'm not sure I've seen the tests running:
https://github.com/jfrog/jfrog-vscode-extension/actions/runs/4366198697/jobs/7635874087